modular_azurepeak/code/game/objects/items/ritualcircles.dm

```diff
@@ -306,7 +306,7 @@
 
 /obj/structure/ritualcircle/eora
```

Comment:
Eora can be reflavored into Gani-adjacent stuff. Eora is the designated ERP god on Roguecode offshoots, so in an effort to make them _not_ be that, we'll go to the next most pacificist/loving patron, being Gani of course.

Reply:
That totally works, but I'm fearful of making the Rune of Love spells related to Gani because some of the things that spell does are downright evil looking.